checks/cs50/2017/x/mario/mario.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Mario(Checks):
+    @check()
+    def exists(self):
+        """mario.c exists."""
+        super(Mario, self).exists("mario.c")
+
+    @check("exists")
+    def compiles(self):
+        """mario.c compiles."""
+        self.spawn("clang -o mario mario.c -lcs50").exit(0)
+
+    @check("compiles")
+    def test_reject_negative(self):
+        """rejects a height of -1"""
+        self.spawn("./mario").stdin("-1").reject().kill()
```

Comment:
Is `kill` necessary, or could our post-check teardown just kill and still-open `Child`?